new-feature.txt

```diff
@@ -0,0 +1 @@
+This file was created in feature-1 branch
```

Comment:
Very nice and important line of code!:-)

Reply:
Thanks for warm words